mender-convert-modify

```diff
@@ -225,19 +225,34 @@ EOF
 log_info "Installing Mender client and related files"
 
 if [ "${MENDER_CLIENT_INSTALL}" = "y" ]; then
+    MENDER_CLIENT_DEB_VERSION="$MENDER_CLIENT_VERSION"
+    if [ "$MENDER_CLIENT_VERSION" = "auto" ]; then
+        deb_distro=$(probe_debian_distro_name)
+        deb_codename=$(probe_debian_distro_codename)
+        if ([ "$deb_distro" = "debian" ] && [ "$deb_codename" = "bullseye" ]) \
+            || ([ "$deb_distro" = "debian" ] && [ "$deb_codename" = "buster" ]) \
+            || ([ "$deb_distro" = "ubuntu" ] && [ "$deb_codename" = "jammy" ]) \
+            || ([ "$deb_distro" = "ubuntu" ] && [ "$deb_codename" = "focal" ]); then
+            MENDER_CLIENT_VERSION="3.5.x"
```

Comment:
I am missing something, how is this supposed to work? We won't be able to download a 3.5.**x** package, we need somewhere to translate this into a tag.

I think rather you want to select on package name, like for this case `mender-client` and in the _else_ below the `mender-client4` package.

Reply:
@lluiscampos 3.5.x will download the latest version of the `mender-client` package; in fact, anything which starts with "3." will trigger the first stanza of the if. The effect is "latest" on the "mender-client" package.

Reply:
You are right - I didn't pay attention to that we are already installing `mender-auth` and `mender-udpate` below.